zippicks-master-critic-v2/includes/class-master-critic.php

```
@@ -0,0 +1,92 @@
+        
+        // Public (Frontend)
+        require_once ZPMC_PLUGIN_DIR . 'public/class-public.php';
+        
+        // Admin
+        require_once ZPMC_PLUGIN_DIR . 'admin/class-admin.php';
+    }
+    
+    private function define_hooks() {
+        // Initialize classes
+        $plugin_admin = new ZipPicks_Master_Critic_Admin($this->get_plugin_name(), $this->get_version());
+        $plugin_public = new ZipPicks_Master_Critic_Public($this->get_plugin_name(), $this->get_version());
```

PHPMD (2.15.0)

Avoid unused local variables such as '$plugin_public'. (Unused Code Rules)

(UnusedLocalVariable)